lib/Importer/Caffe2.cpp

```diff
@@ -415,6 +447,65 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
+  if (typeName == "Tanh") {
```

Comment:
Can we have this and other ops in CommonOperatorLoader? I think they have the same semantics in ONNX.

Reply:
I can move it/others, but I have no ONNX proto with these operators to make sure they work correctly, so I was conservatively adding only for C2.

Reply:
I moved all but Slice -- there is a slight difference in the ONNX/C2 implementations for the start and end indices and negative values, so I'd rather not add it in without having a proto with it to test that it works.